CHANGELOG.rst

```diff
@@ -0,0 +1,9 @@
+=========
+CHANGELOG
+=========
+
+
+0.1.0 (UNRELEASED)
```

Comment:
Can we find a better description than *Unreleased*? Say, *In progress*, or *To be released*.